.github/workflows/mariner2.yml

```diff
@@ -0,0 +1,45 @@
+name: build for mariner2
+
+on:
+  push
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l -o pipefail {0}
+    container:
+      image: mcr.microsoft.com/cbl-mariner/base/core:2.0
+    steps:
+      - run: |
+          tdnf install --noplugins --skipsignature -y ca-certificates git
+      - name: checkout
+        uses: actions/checkout@v4
+        with:
+          submodules: 'recursive'
+          set-safe-directory: true
+      - name: install dependencies
+        run: |
+          bash .install/mariner2.sh
+      - name: get redis latest
+        run: |
+          bash .install/get_redis.sh
+      - name: build
+        run: |
+          make build
+      - name: test
+        run: |
+          make test
+      - name: pack module
+        run: |
+          make pack BRANCH=${{ github.ref_name }}
+      - name: get aws
+        # TODO: New script .install/get_aws.sh?
```

Comment:
I just added the `aws` install to the dependencies script

Reply:
OK. I did the same change. Thanks.